extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_dropdown.php

```diff
@@ -0,0 +1,24 @@
+<div class="wds-global-navigation__<?= Sanitizer::escapeClass( $type ); ?> wds-dropdown wds-is-active">
+	<div class="wds-dropdown-toggle wds-global-navigation__dropdown-toggle">
+		<?php if ( $model['header']['type'] === 'line-text' ): ?>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "header" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)